drivers/virt/nitro_enclaves/ne_misc_dev.c

```diff
@@ -140,7 +140,7 @@ static int user_access_ok(void __user *addr, unsigned long size)
 #endif
 }
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 7, 0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 4, 0)
```

Comment:
These kernel version markers are set to follow the upstream Linux kernel, as in which version from upstream, the functionality has been included. The main source for the Nitro Enclaves kernel driver is the upstream Linux kernel.

If still needed, this out-of-tree codebase from the drivers folder could be used for the systems that do not have the NE kernel driver already integrated. Amazon Linux 2 already has the NE kernel driver included in their kernels. One option could be to comment / remove these lines, if needed to test / debug using this source code. 

Open to discuss if certain use cases would need to consider other options for this out-of-tree codebase.

Reply:
Ok, got it. 

> One option could be to comment / remove these lines, if needed to test / debug using this source code.

I initially removed these lines to test locally, but found it a bit weird, so decided to change the required kernel version. I'll drop this commit. Thanks for the explanation!